pkg/environments/gitops.go

```diff
@@ -51,87 +63,20 @@ type EnvironmentPullRequestOptions struct {
 // the message as the body for both the commit and the pull request,
 // and the pullRequestInfo for any existing PR that exists to modify the environment that we want to merge these
 // changes into.
-func (o *EnvironmentPullRequestOptions) Create(env *jenkinsv1.Environment, branchNameText *string, title *string,
-	message *string, environmentsDir string, pullRequestInfo *gits.PullRequestInfo) (*gits.PullRequestInfo, error) {
-	var answer *gits.PullRequestInfo
-	source := &env.Spec.Source
-	gitURL := source.URL
-	if gitURL == "" {
-		return answer, fmt.Errorf("No source git URL")
-	}
-	gitInfo, err := gits.ParseGitURL(gitURL)
-	if err != nil {
-		return answer, err
-	}
-
-	if err != nil {
-		return answer, err
-	}
-	dir := filepath.Join(environmentsDir, gitInfo.Organisation, gitInfo.Name)
+func (o *EnvironmentPullRequestOptions) Create(env *jenkinsv1.Environment, environmentsDir string,
```

Comment:
Is is not really apps related right? It is a fallout of the additional structs you introduce, right?

Reply:
It's part of a more generic piece of functionality that allows us to create PRs against GitOps environments. The `dev` environment is where we store info on installed apps when we're using gitops mode.

Reply:
I see